README.md

```diff
@@ -14,6 +14,14 @@ Integrate chat completion into your business applications with SAP Cloud SDK for
 
 _Insert a short description what is required to get your project running..._
 
+# 📚 Documentation
+- [AI Core APIs](./docs/ai-core-APIs.md)
```

Comment:
[pp] Maybe have a section called `packages`, list the package name `@sap-ai-sdk/ai-core` and a short description of what the specific package does.